Challenge1-Vite/src/components/Filters.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+
+interface FiltersProps {
+  searchTerm: string;
+  setSearchTerm: (term: string) => void;
+  setSortOption: (option: string) => void;
+  setFilterFavorites: (isFavorite: boolean) => void;
+}
+
+const Filters: React.FC<FiltersProps> = ({ searchTerm, setSearchTerm, setSortOption, setFilterFavorites }) => {
+  const [isFavorites, setIsFavorites] = useState(false);
+  const [dropdownOpen, setDropdownOpen] = useState(false);
```

Comment:
Not super important, but try to be consistent with naming, if the first state is named `isFavorites`, then the second one should be named `isDropdownOpen`